packages/taglib-windows.0.3.4/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-taglib"
+build: [
+  ["./configure" "--host=%{conf-gcc-windows:host}%" "--prefix=%{prefix}%" "OCAMLFIND_TOOLCHAIN=windows"]
+  [make]
+]
+install: [
+  ["env" "OCAMLFIND_TOOLCHAIN=windows"
+   "make" "install"]
```

Comment:
Not a blocker here but we should consistently use either `"make"` or `make`, for the sake of OS X (where `make` expands/may expand to `"gmake"` under some conditions I am not entirely clear on).

Reply:
Good point. It's a mistake carried over from the upstream packages, will fix in the future.